ofctrl/ofSwitch.go

```diff
@@ -32,7 +32,9 @@ import (
 )
 
 const (
-	messageTimeout = 10 * time.Second
+	// The default BUNDLE_IDLE_TIMEOUT is 10s and we need wait a bit
+	// longer to overcome the timer difference.
+	messageTimeout = 13 * time.Second
```

Comment:
Does it still need to be updated?

Reply:
Since the timer will be slightly different I prefer to wait longer here.
But no strong opinion here.